### PR TITLE
Added 'commands' and 'man' to document the bot's behavior

### PR DESCRIPTION
**Pull request by AlexKolpa**
So every new kid in the channel will finally figure what the bot can do.

### CODE REVIEW

File: bot.js [hunk 1]
```diff
@@ -145,28 +145,92 @@ client.on('join', function (channel, user) {
 
 client.on('message', function (from, channel, message) {
 	var activatedPlugins = config.plugins;
+
 	if (channel[0] === '#') {
 		activatedPlugins = state[channel].plugins;
 	}
 
-	if (message[0] === '!') {
+	if (message.trim() === '!commands') {
```
Comment (Sebmaster) [inline bot.js]: I think we shouldn't trim here, we also don't trim later on with the message[0] check.

Comment (AlexKolpa) [inline bot.js]: But [we do](https://github.com/Sebmaster/TwoDee-Bot/pull/99/files#diff-9d199097bf1be7e582b7cb3a378cfbf0R157), and [we did](https://github.com/Sebmaster/TwoDee-Bot/pull/99/files#diff-9d199097bf1be7e582b7cb3a378cfbf0L158)

Comment (Sebmaster) [inline bot.js]: That's only the arguments for a command however, if you space before the !, the bot will not react.

Comment (AlexKolpa) [inline bot.js]: Good point. Fixed it to make consistent.

**Comment by Sebmaster** [comment]
Merged as 80d73c89460f5ab8c5678ad09c3a07e0bfde81ae.

**Pull request closed without merging**